src/main.cpp

```diff
@@ -4634,6 +4619,12 @@ bool ReceivedBlockTransactions(
             for (const auto& out : tx.vout) {
                 transparentValueDelta += out.nValue;
             }
+
+            for (auto elem : chainparams.GetConsensus().GetActiveFundingStreamElements(pindexNew->nHeight)) {
+                if (std::holds_alternative<Consensus::Lockbox>(elem.first)) {
+                    lockboxValueDelta += elem.second;
+                }
+            }
```

Comment:
It's not necessary to support forks that activate a lockbox in the genesis block.
```suggestion
            assert(chainparams.GetConsensus().GetActiveFundingStreamElements(pindexNew->nHeight).size() == 0);
```
If there is ever such a fork (unlikely), they will hit this assertion and know what to change.